src/driving_distance/many_to_dist_withPointsDD.c

```diff
@@ -54,6 +54,9 @@ void process(
         Path_rt **result_tuples,
         size_t *result_count) {
     driving_side[0] = estimate_drivingSide(driving_side[0]);
+    if (driving_side[0] != 'r' && driving_side[0] != 'l') {
```

Comment:
Probably not the best place to check, did you analize what `estimate_drivingSide` is doing?

Reply:
Yes, I did. `estimate_drivingSide` converts the input character to lowercase, and returns `r`, `l` or `b` (if it is neither `r` nor `l`)
As the `estimate_drivingSide` function is also used by other withPoints functions, modifying it will have a greater impact, 
so I made modifications above with reference to the following code: https://github.com/pgRouting/GSoC-pgRouting/blob/81bcd0c65fcd127b9497cda5375e0800b7ec7ecc/src/trsp/trsp_withPoints.c#L68-L71

 But if the default for versions 3.6 and after is `r`, maybe I can change `estimate_drivingSide`?

Reply:
See #300